Makefile

```diff
@@ -638,6 +638,10 @@ generate-go-conversions: $(CONVERSION_GEN) ## Generate conversions go code
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate.go.txt
 
+.PHONY: generate-package-secret ## Generate the default pinniped addon secret. e.g. make generate-package-secret TKR_VERSION=v1.23.3---vmware.1-tkg.1 IAAS=vsphere
```

Comment:
here is the analogous pr for generating package config's (e.g., `AntreaConfig`): https://github.com/vmware-tanzu/tanzu-framework/pull/1857. looks like that pr uses lowercase symbols for their `make` variables (e.g., `tkr`, `kind`, etc.).

i saw some discussion on this previously, but figured i'd loop folks in here.

@ZihanJiang96 @blc1996 - what `make` variable should we be using here? i saw https://github.com/vmware-tanzu/tanzu-framework/pull/1857 uses `tkr`, but we were told to use `TKR_VERSION`. i think we should use the same across the two `make` targets (`generate-package-secret`, `generate-package-config`).

Reply:
Given [PR for generate-package-config](https://github.com/vmware-tanzu/tanzu-framework/pull/1857/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R761) I'll update to `${tkr}` and `${iaas}`